src/sandstorm/appid-replacements.capnp

```diff
@@ -110,7 +110,17 @@ const appIdReplacementList :List(AppIdReplacement) = [
   # See also [this discussion](https://github.com/wekan/wekan/issues/640) about the future
   # governance of Wekan.
 
-# ---- end Wekan entry ----
+  # ---- end Wekan entry ----
+  
+  # ---- WordPress entry ----
+
+  (original = "aax9j672p6z8n7nyupzvj2nmumeqd4upa0f7mgu8gprwmy53x04h",
+   replacement = "zp00qqrha6gp8515ju84ys91syyq8ezv77g9s1r0t3urz92uzmqh"),
+  # The original key is held by Jan Jambor and likely members of the original Sandstorm team. It's
+  # been half a decade since this app was updated, so I don't want to block on requesting a key
```

Comment:
Nit:

- Prose that mentions relative spans of time without reference to when it was written is likely to bitrot; maybe mention the date this change is being made, so future readers are not wondering when "now" is.
- Similarly, I try to avoid using the first person without making it clear who the author of the comment is; I would either add your name in parens or rephrase this to be more of a group decision (which it will be, if we all decide this should be merged).